src/databricks/labs/remorph/assessments/pipeline.py

```diff
@@ -45,10 +45,14 @@ def _execute_sql_step(self, step: Step):
 
         # Execute the query using the database manager
         logging.info(f"Executing query: {query}")
-        result = self.executor.execute_query(query)
+        try:
```

Comment:
Maybe rename the PR since this isn't just a test ?

Reply:
Done